app/renderers/post_link_renderer.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+class PostLinkRenderer < WillPaginate::ActionView::LinkRenderer
+  def previous_or_next_page(page, text, classname)
+    if page
+      submit(text, page, classname, class: classname)
+    else
+      tag(:span, text, class: classname + ' disabled')
+    end
+  end
+
+  def page_number(page)
+    if page == current_page
+      tag(:em, page, class: 'current')
```

Comment:
This is one of the differences with our current **PaginationListLinkRenderer**: the existing one renders `span`s instead of `em`s, so it would be nice if this one did too so that the post pagination looks the same as the pagination we have elsewhere on the site (but maybe have a look at our existing renderer and see if there are any other differences too)